### ISSUE TITLE
A11y_Azure Tools for VS Code_CreateNewProject_SelectTemplatForYourProjectFirstFunction_Screenreader: NVDA is announcing the name of dialog box when user opens it.

### ISSUE THREAD

**Issue by sohelmulani1998** [microsoft/vscode-azurefunctions#3123]
**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #Azure Tools for VS Code; #BM_Azure Tools for VS Code_Win32_Apr22; #Win32; #FTP; #JAWS; #NVDA; #Screenreader;

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**

1. Open Visual studio code
2. Tab till extension button and search for "Azure Functions"
3. Tab till install and press enter now tab till the "Azure" button and press enter.
4. Turn on the NVDA.
5. Tab till Create 'Create New Project'.
6. Check is NVDA is announcing the name of dialog box that appears.

**Actual:**
NVDA is announcing the name of dialog box when user opens it.
Note: 1) This issue is only producing with NVDA and not with Jaws.
          2)This issue is producing for all the dialog box that appears on the screen.

**Expected Result:**
NVDA should be able to announce the name of dialog box when it opens.

**Attachment:**
![500eb5a6-e802-4474-9395-331f5852b0d9](https://user-images.githubusercontent.com/85861804/164405092-3dc3ecaf-46a8-4747-b641-7f4666a2728b.gif)

**Comment by MicroFish91** [comment]
Opened an issue with VS Code team which should cover this: https://github.com/microsoft/vscode/issues/149387

**Comment by MicroFish91** [comment]
This [comment](https://github.com/microsoft/vscode-azurefunctions/issues/2942#issuecomment-1289787198) applies here as well, same upstream tracking.

**Comment by sohelmulani1998** [comment]
Downgrading the severity as per New Severity guidelines.

**Comment by MicroFish91** [comment]
I believe the upstream issue for this one has been resolved, can you please confirm and close if so?  Thank you!
